lang/python/python-awscli/Makefile

```diff
@@ -0,0 +1,79 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=awscli
+PKG_VERSION:=1.16.75
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+PKG_HASH:=8d96ec0de325ea8271cc6aa95b7392bbf548ec4aabd3ffbcdc0619b64edd4a45
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/a/awscli
+PKG_MAINTAINER:=Daniel Danzberger <daniel@dd-wrt.com>
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-awscli-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+include ../python-package.mk
+include ../python3-package.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/python-awscli/Default
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=awscli
+  URL:=https://pypi.org/project/awscli/
+endef
+
+define Package/python-awscli
+$(call Package/python-awscli/Default)
+  DEPENDS:=+python \
+	+python-six \
+	+python-yaml \
+	+python-pyasn1 \
+	+python-botocore \
+	+python-rsa \
+	+python-colorama \
+	+python-futures \
+	+python-docutils \
+	+python-jmespath \
+	+python-dateutil \
+	+python-s3transfer \
+	+python-simplejson
```

Comment:
I was looking at the setup requirements for [awscli](https://github.com/aws/aws-cli/blob/1.16.75/setup.py#L26-L31) and botocore (parts [1](https://github.com/boto/botocore/blob/1.12.66/setup.py#L26-L27), [2](https://github.com/boto/botocore/blob/1.12.66/setup.py#L43), [3](https://github.com/boto/botocore/blob/1.12.66/setup.py#L50)), and it looks like some of botocore's dependencies (jmespath, dateutil) are assigned to `python-awscli` instead of `python-botocore`.

Can you double check the dependencies of each package to ensure they are assigned correctly?

Reply:
yes, there were some more dependencies for botocore.